examples/js/postprocessing/EffectComposer.js

```diff
@@ -203,3 +203,39 @@ Object.assign( THREE.Pass.prototype, {
 	}
 
 } );
+
+// Helper for passes that need to fill the viewport with a single quad.
+THREE.Pass.FullScreenQuad = function ( material ) {
+
+	this._mesh = new THREE.Mesh( THREE.Pass.FullScreenQuad._geometry, material );
+
+};
+
+Object.defineProperty( THREE.Pass.FullScreenQuad.prototype, 'material', {
+
+	get: function () {
+
+		return this._mesh.material;
+
+	},
+
+	set: function ( value ) {
+
+		this._mesh.material = value;
+
+	}
+
+} );
+
+Object.assign( THREE.Pass.FullScreenQuad.prototype, {
+
+	render: function ( renderer ) {
+
+		renderer.render( this._mesh, THREE.Pass.FullScreenQuad._camera );
+
+	}
+
+} );
+
+THREE.Pass.FullScreenQuad._camera = new THREE.OrthographicCamera( - 1, 1, 1, - 1, 0, 1 );
+THREE.Pass.FullScreenQuad._geometry = new THREE.PlaneBufferGeometry( 2, 2 );
```

Comment:
We should consider to put the entire `THREE.Pass` definition into a closure. In this way, it's not necessary to pollute the `FullScreenQuad` namespace with private variables.

Reply:
I hid the _camera and _geometry in a closure now.

If this is expected it would be good if the code style notes would mention something about it.